doc/source/user_guide/variational_quantum_algorithms.rst

```diff
@@ -0,0 +1,30 @@
+Variational Quantum Algorithms with TensorLy-Quantum
```

Comment:
To replace maxcut with a new file, you also have to update user_guide/index.rst to link to the new file instead of `maxcut`